packages/config/esbuild/static/cli.js

```diff
@@ -0,0 +1,14 @@
+const cli = {
+  bundle: true,
+  entryPoints: [
+    './src/main.ts',
```

Comment:
shouldn't this be index,js?

Reply:
Not for `CLI`, cli needs a different entry point because it's an IEF [immediately executing file].